### PR TITLE
[Java] Add Unit Tests for DEBUG Log Level

### PR DESCRIPTION
**Pull request by lixin-wei**
## Why are these changes needed?

Add unit tests for DEBUG log level

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by jovany-wang — APPROVED** [review]
NICE PR！

**Comment by jovany-wang** [comment]
Test stilll fails.

**Comment by jovany-wang** [comment]
I'll debug these failures later.

File: src/ray/raylet/worker_pool.cc [hunk 1]
```diff
@@ -256,25 +256,20 @@ Process WorkerPool::StartWorkerProcess(
...
                       std::to_string(workers_to_start));
-  }
-
-  // Append startup-token for JAVA here
-  if (language == Language::JAVA) {
+    // Append startup-token for JAVA here
     options.push_back("-Dray.raylet.startup-token=" +
                       std::to_string(worker_startup_token_counter_));
+                      
+    options.push_back("-Dray.job.logging-level=" + job_config->logging_level());
```
Comment (jovany-wang) [inline src/ray/raylet/worker_pool.cc:272]:
```suggestion
    options.push_back("-Dray.logging-level=" + job_config->logging_level());
```

**Comment by jovany-wang** [comment]
`testDebugLogLevel` passed. 
This PR can be merged once CI green.

**Comment by jovany-wang** [comment]
@lixin-wei Could you retry the CI or merge the master?

**Comment by bveeramani** [comment]
## :bangbang: ACTION REQUIRED :bangbang:

We've switched our code formatter from YAPF to Black (see #21311).

To prevent issues with merging your code, here's what you'll need to do:

1. Install Black
```
pip install -I black==21.12b0
```

2. Format changed files with Black
```
curl -o format-changed.sh https://gist.githubusercontent.com/bveeramani/42ef0e9e387b755a8a735b084af976f2/raw/7631276790765d555c423b8db2b679fd957b984a/format-changed.sh
chmod +x ./format-changed.sh
./format-changed.sh
rm format-changed.sh
```

3. Commit your changes.
```
git add --all
git commit -m "Format Python code with Black"
```

4. Merge master into your branch.
```
git pull upstream master
```

5. Resolve merge conflicts (if necessary).

After running these steps, you'll have the updated `format.sh`.

**Comment by kfstorm** [comment]
## :bangbang: ACTION REQUIRED :bangbang:

We've updated our formatting configuration for C++ code. (see #22725)

This PR includes C++ code change. To prevent issues with merging your code, here's what you'll need to do:

1. Merge the latest changes from upstream/master branch into your branch.

```bash
git pull upstream master
git merge upstream/master
```

2. Resolve merge conflicts (if necessary).

After running these steps, you'll have the updated C++ formatting configuration.

3. Format changed files.

```bash
scripts/format.sh
```

4. Commit your changes.

```bash
git add --all
git commit -m "Format C++ code"
```

**Pull request closed without merging**